internal/ci/vulnerability_result_diff_test.go

```diff
@@ -67,6 +67,14 @@ func TestDiffVulnerabilityResults(t *testing.T) {
 			// opposite of above test case, result should be empty
 			wantPath: "fixtures/vulns/test-vuln-diff-b-c.json",
 		},
+		{
+			args: args{
+				oldRes: testutility.LoadJSONFixture[models.VulnerabilityResults](t, "fixtures/vulns/test-vuln-results-a.json"),
+				newRes: testutility.LoadJSONFixture[models.VulnerabilityResults](t, "fixtures/vulns/test-vuln-results-d.json"),
+			},
+			// `newRes` has a package that is not in `oldRes` and has no vulns, so result should just be empty
+			wantPath: "fixtures/vulns/test-vuln-diff-b-c.json",
```

Comment:
@another-rex its not a big deal, but it feels like we could use snapshots to handle the "want" path rather than needing to have dedicated files?